modules/navier_stokes/test/tests/finite_volume/ins/channel-flow/SIMPLEC/2d/2d-velocity-pressure.i

```diff
@@ -0,0 +1,168 @@
+    ix = '20'
+    iy = '3'
+  []
+[]
+
+[Problem]
+  linear_sys_names = 'u_system v_system pressure_system'
+  previous_nl_solution_required = true
+[]
+
+[UserObjects]
+  [rc]
+    type = RhieChowMassFlux
+    u = vel_x
+    v = vel_y
+    pressure = pressure
+    rho = ${rho}
+    p_diffusion_kernel = p_diffusion
+    pressure_projection_method = SIMPLEC
+  []
+[]
+
+[Variables]
+  [vel_x]
+    type = MooseLinearVariableFVReal
+    initial_condition = 0.5
+    solver_sys = u_system
+  []
+  [vel_y]
+    type = MooseLinearVariableFVReal
+    solver_sys = v_system
+    initial_condition = 0.0
+  []
+  [pressure]
+    type = MooseLinearVariableFVReal
+    solver_sys = pressure_system
+    initial_condition = 0.2
+  []
+[]
+
+[LinearFVKernels]
+  [u_advection_stress]
+    type = LinearWCNSFVMomentumFlux
+    variable = vel_x
+    advected_interp_method = ${advected_interp_method}
+    mu = ${mu}
+    u = vel_x
+    v = vel_y
+    momentum_component = 'x'
+    rhie_chow_user_object = 'rc'
+    use_nonorthogonal_correction = false
+  []
+  [v_advection_stress]
+    type = LinearWCNSFVMomentumFlux
+    variable = vel_y
+    advected_interp_method = ${advected_interp_method}
+    mu = ${mu}
+    u = vel_x
+    v = vel_y
+    momentum_component = 'y'
+    rhie_chow_user_object = 'rc'
+    use_nonorthogonal_correction = false
+  []
+  [u_pressure]
+    type = LinearFVMomentumPressure
+    variable = vel_x
+    pressure = pressure
+    momentum_component = 'x'
+  []
+  [v_pressure]
+    type = LinearFVMomentumPressure
+    variable = vel_y
+    pressure = pressure
+    momentum_component = 'y'
+  []
+  [p_diffusion]
+    type = LinearFVAnisotropicDiffusion
+    variable = pressure
+    diffusion_tensor = Ainv
+    use_nonorthogonal_correction = false
+  []
+  [HbyA_divergence]
+    type = LinearFVDivergence
+    variable = pressure
+    face_flux = HbyA
+    force_boundary_execution = true
+  []
+[]
+
+[LinearFVBCs]
+  [inlet-u]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'left'
+    variable = vel_x
+    functor = '1.1'
+  []
+  [inlet-v]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'left'
+    variable = vel_y
+    functor = '0.0'
+  []
+  [walls-u]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'top bottom'
+    variable = vel_x
+    functor = 0.0
+  []
+  [walls-v]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'top bottom'
+    variable = vel_y
+    functor = 0.0
+  []
+  [outlet_p]
+    type = LinearFVAdvectionDiffusionFunctorDirichletBC
+    boundary = 'right'
+    variable = pressure
+    functor = 1.4
+  []
+  [outlet_u]
+    type = LinearFVAdvectionDiffusionOutflowBC
+    variable = vel_x
+    use_two_term_expansion = false
+    boundary = right
+  []
+  [outlet_v]
+    type = LinearFVAdvectionDiffusionOutflowBC
+    variable = vel_y
+    use_two_term_expansion = false
+    boundary = right
+  []
+[]
+
+[Executioner]
+  type = SIMPLE
```

Comment:
It would be kind of nice to not have a discrepancy between the executioner name and pressure projection method, but maybe experienced CFD users wouldn't find this confusing?

Reply:
I like this design. The projection type only comes in the picture through the fields that are computed in the RC object.

Reply:
I agree the code design is nice

Reply:
Thanks. Keeping this design. Will change the `pressure_projection_method` to `consistent` for SIMPLEC ot `inconsistent` for simple to avoid confusions